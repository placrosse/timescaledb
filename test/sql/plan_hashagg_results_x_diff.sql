-- Copyright (c) 2016-2018  Timescale, Inc. All Rights Reserved.
--
-- This file is licensed under the Apache License,
-- see LICENSE-APACHE at the top level directory.

\set ECHO errors
\o /dev/null
SET client_min_messages = 'error';
\ir include/plan_hashagg_load.sql
RESET client_min_messages;
\o

--generate the results into two different files
SET client_min_messages = 'error';
\set ECHO none
--make output contain query results
\set PREFIX ''
\o :TEST_OUTPUT_DIR/results/plan_hashagg_optimized_results.out
SET timescaledb.disable_optimizations= 'off';
\ir include/plan_hashagg_query.sql
\o
\o :TEST_OUTPUT_DIR/results/plan_hashagg_unoptimized_results.out
SET timescaledb.disable_optimizations= 'on';
\ir include/plan_hashagg_query.sql
\o
RESET client_min_messages;

\! diff ${TEST_OUTPUT_DIR}/results/plan_hashagg_optimized_results.out ${TEST_OUTPUT_DIR}/results/plan_hashagg_unoptimized_results.out

SELECT 'Done';
