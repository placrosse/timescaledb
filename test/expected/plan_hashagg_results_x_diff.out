-- Copyright (c) 2016-2018  Timescale, Inc. All Rights Reserved.
--
-- This file is licensed under the Apache License,
-- see LICENSE-APACHE at the top level directory.
\set ECHO errors
psql:include/plan_hashagg_query.sql:61: ERROR:  timestamp units "invalid" not recognized
psql:include/plan_hashagg_query.sql:61: ERROR:  timestamp units "invalid" not recognized
 ?column? 
----------
 Done
(1 row)

